### PR TITLE
Update CONTRIBUTING.rst

### PR DESCRIPTION
**Pull request by abhi27-web**
Fixes #2375 

### Tasks

* [x] Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] Tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
  Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors. 
* [x] Commits are in uniquely-named feature branch and has no merge conflicts 📁

Signed-off-by: Abhigya Verma <abhigyaverma27@gmail.com>

### CODE REVIEW

**Comment by sritasngh** [comment]
Hey add signed off in the commit message body:)

**Comment by sritasngh** [comment]
> Fixes #0000
> 
> ### Tasks
> * [x]  Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
> * [x]  PR is descriptively titled bookmark_tabs and links the original issue above link
> * [x]  Tests pass -- look for a green checkbox heavy_check_mark a few minutes after opening your PR
>   Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors.
> * [x]  Commits are in uniquely-named feature branch and has no merge conflicts file_folder

and here to signed off and issue no.

**Comment by itssingh** [comment]
Also see the issue repo, it has been created in the aboutcode https://github.com/nexB/aboutcode/issues... so you merge request should be to aboutcode master branch:)

**Comment by abhi27-web** [comment]
> Hey add signed off in the commit message body:)

Done

**Comment by abhi27-web** [comment]
I am really new to all this can you guide me how to edit a commit message body?

**Review by abhi27-web — COMMENTED** [review]
Signed-off-by: Abhigya Verma <abhigyaverma@gmail.com>

**Comment by abhi27-web** [comment]
Signed-off-by: Abhigya Verma <abhigyaverma27@gmail.com> 

Issue no. #2375

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ with your name and email::
     Signed-off-by: Jane Doe <developer@example.com>
```
Comment (sritasngh) [inline CONTRIBUTING.rst:22]: You will have to remove this signed-off too

**Review by sritasngh — CHANGES_REQUESTED** [review]

**Comment by sritasngh** [comment]
> Signed-off-by: Abhigya Verma [abhigyaverma27@gmail.com](mailto:abhigyaverma27@gmail.com)
> 
> Issue no. #2375

Instead edit the top comment:)

**Comment by sritasngh** [comment]
> Fixes #0000
> 
> ### Tasks
> * [x]  Reviewed [contribution guidelines](https://github.com/nexB/scancode-toolkit/blob/develop/CONTRIBUTING.rst)
> * [x]  PR is descriptively titled bookmark_tabs and links the original issue above link
> * [x]  Tests pass -- look for a green checkbox heavy_check_mark a few minutes after opening your PR
>   Run [tests](https://scancode-toolkit.readthedocs.io/en/latest/contribute/contrib_dev.html#running-tests) locally to check for errors.
> * [x]  Commits are in uniquely-named feature branch and has no merge conflicts file_folder
> 
> Signed-off-by: Abhigya Verma [abhigyaverma27@gmail.com](mailto:abhigyaverma27@gmail.com)
> 
> Issue No. #2375

This one

File: CONTRIBUTING.rst (hunk 1 above)
Comment (sritasngh) [reply CONTRIBUTING.rst:22]: oops sorry, I guess this is the part of doc.

Comment (abhi27-web) [reply CONTRIBUTING.rst:22]: yes

Comment (sritasngh) [reply CONTRIBUTING.rst:22]: no need to change.

**Comment by abhi27-web** [comment]
Is everything ok now?

File: CONTRIBUTING.rst (hunk 1 above)
Comment (abhi27-web) [reply CONTRIBUTING.rst:22]: cool

**Comment by sritasngh** [comment]
Go through contribution guidelines once, you will get idea about writing good commit msg and pr comment and others things too:)

**Comment by abhi27-web** [comment]
> Go through contribution guidelines once, you will get idea about writing good commit msg and pr comment and others things too:)

This means right now my pull request is not being merged?

**Comment by sritasngh** [comment]
> > Go through contribution guidelines once, you will get idea about writing good commit msg and pr comment and others things too:)
> 
> This means right now my pull request is not being merged?

haha, be patient, Merge is a long process:)

**Comment by abhi27-web** [comment]
> > > Go through contribution guidelines once, you will get idea about writing good commit msg and pr comment and others things too:)
> > 
> > 
> > This means right now my pull request is not being merged?
> 
> haha, be patient, Merge is a long process:)

Ok but I dont need to do anything right?

**Comment by abhi27-web** [comment]
What is the update??

**Comment by pombredanne** [comment]
This has been merged with: https://github.com/nexB/scancode-toolkit/commit/5c7208aa2d81b02398da6bab5b37d2751b341b1e
Thanks!

**Pull request closed without merging**